docs/KNOWN_ISSUES.md

```diff
@@ -20,20 +20,6 @@ guest_customization/gosc_cloudinit_staticip.yml
 * Workaround: Please wait for the fix, or make sure not deleting VMs while the test is running.
 * Affected test cases: Testing will fail if any of the task failed due to this issue.
 
-## VMware tools
-1. open-vm-tools 11.1.0 and later
-* Failure: Error logs in 'var/log/vmware-imc/toolsDeployPkg.log' of Perl guest OS customization in RHEL/CentOS/Oracle Linux 7.x and 8.x, e.g.,
-```
-[2021-03-18T15:52:10.974Z] [   error] Customization command failed with exitcode: 127, stderr: ''.
-[2021-03-18T15:52:11.974Z] [   error] Failed to reboot, telinit returned error 127.
```

Comment:
This "failed to reboot" error message is not an error actually?

Reply:
Even though we see this message in toolsDeployPkg.log, it doesn't mean that the guest was not rebooted. By checking with Pengpeng, he said that as long as all customized properties are successfully set in guest. We can consider that GOSC succeeds. At present, if 'uncaught signal 15' was reported in toolsDeployPkg.log, it is an real error(see PR 2716292). Maybe I can add checking 'uncaught signal 15' and record this as a known issue.